src/plugins/expandOperation.ts

```diff
@@ -0,0 +1,98 @@
+import { Host, Session } from "@azure-tools/autorest-extension-base";
+import { CodeModel, Request, Operation, Parameter } from "@azure-tools/codemodel";
+import { isNullOrUndefined } from "util";
+import { Helper } from "../helper";
+import { CliConst, CliCommonSchema } from "../schema";
+import { NodeHelper } from "../nodeHelper";
+import { Modifier } from "./modifier/modifier";
+import { CopyHelper } from "../copyHelper";
+
+export class ExpandOperation{
+
+    constructor(protected session: Session<CodeModel>){
+    }
+
+    public async process() {
+
+        await this.modifier();
+
+        for (const group of this.session.model.operationGroups) {
+            const existedNames = new Set<string>(group.operations.map((op) => op.language.default.name.toLowerCase()));
+            const expandedGroupOperations = [];
+            for (const operation of group.operations) {
+                const expandNames = NodeHelper.getCliExpandOperationNames(operation);
+                if (!expandNames) {
```

Comment:
>expandNames [](start = 21, length = 11)

or exampleNames.length === 0. 

is ! cover null and undefined? always confused about this in js ...

Reply:
You are right. thanks